swap/swap_test.go

```diff
@@ -1103,8 +1103,14 @@ func testCashCheque(s *Swap, otherSwap cswap.Contract, opts *bind.TransactOpts,
 }
 
 // newDefaultParams creates a set of default params for tests
-func newDefaultParams() *Params {
+func newDefaultParams(t *testing.T) *Params {
```

Comment:
If you pass a `testing.T` we should use `t.Helper()`

Reply:
i agree, fixed